OPTIONS.md

```diff
@@ -0,0 +1,46 @@
+# VarTable Options
+
+## (Required)
+**--input-vcf**		input VCF file
+
+*	Do we need to differentiate between NGS_Mapper VCF (every position) versus lofreq VCF (only variants)?*
+
+**--input-bam**		input BAM file
+
+*	Do we assume BAM index file is in the same directory as BAM or should we include it as an option?
+
+*--input-bai	input BAM index file*
+
+**--input-ref**		input reference FASTA file
+
+*	Assuming we are using a GenBank ref, are we planning to parse annotated data?*
+
+*--accession	input reference fASTA accession number to parse GenBank information*
```

Comment:
Is this feature desirable?

Reply:
The accession flag was my way of addressing my question about a GenBank reference. I'm going to allow the input-ref flag to accept either a reference FASTA (assuming the user has a custom reference or non-annotated reference) or a GenBank extension file (which includes the reference FASTA as well as annotations)